DEVX-2183: Convert "Event Time Processing" recipe to KT (#647)
confluentinc/kafka-tutorials: _includes/tutorials/time-concepts/ksql/markup/answer/short-answer.adoc

> To process data based on *ingestion time*, also known as *log time*, which uses the Kafka record timestamp:

Reviewer: ```suggestion
To process data based on the Kafka record timestamp (either *create time* or *ingestion time*) :
```

Author: @bbejeck I took the spirit of this suggestion and applied it in https://github.com/confluentinc/kafka-tutorials/pull/647/commits/623bcbee8ce6af37b033bc0ca999c64775fdd4b2. WDYT?

+++++
<pre class="snippet"><code class="groovy">{% include_raw tutorials/time-concepts/ksql/code/tutorial-steps/dev/create-stream-logtime.sql  %}</code></pre>
+++++

> To process data based on *event time*, also known as *wall-clock time*, which uses a timestamp from within the record payload:

Reviewer: ```suggestion
To process data using a timestamp from a field within the record payload:
```

Author: @bbejeck I took the spirit of this suggestion and applied it in https://github.com/confluentinc/kafka-tutorials/pull/647/commits/623bcbee8ce6af37b033bc0ca999c64775fdd4b2. WDYT?
